### DISCUSSION TITLE
Do I have to put an item in Daily Notes for it to appear in Day Planner?

### DISCUSSION THREAD

**Discussion by frethop** [ivan-lednev/obsidian-day-planner#428] [Q&A]
The video on the Github README seems to indicate that I can put a "scheduled" flag on a todo list item and Day Planner will pick it up.  But that does not work for me.  

Do I have to put the to do list item in the Day Planner section of Daily Notes?

**Comment by ivan-lednev** [comment]
>  But that does not work for me.

Can you post the steps you followed for that?
